### ISSUE TITLE
Progress bar displays "StringMorph"

### ISSUE THREAD

**Issue by jecisc** [pharo-project/pharo-launcher#669]
I took the development PharoLauncher and when I'm creating a new image (I took the P12 64bit development image) I get some "StringMorph" displayed in the progress bar:

![image](https://github.com/pharo-project/pharo-launcher/assets/9519971/1cc2a9ee-c9f4-4b3d-af77-67b94427627a)

Pharo Launcher version: 3.1.1-404-gef2c9dbc

**Comment by Bajger** [comment]
Relates to PR: https://github.com/pharo-project/pharo-launcher/pull/608 
Probably due to migration from P10 to P11

**Comment by demarey** [comment]
This is strange. I cannot reproduce it even by taking the same version you reported: https://github.com/pharo-project/pharo-launcher/actions/runs/7584839224


<blockquote><img src="https://opengraph.githubassets.com/5ec7b30c102d0a0a24f4475f6e3837842cfd97df8a702fdfffe0892dff4ec105/pharo-project/pharo-launcher" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/pharo-project/pharo-launcher">Allow to define a short name on templates and create an image from th… · pharo-project/pharo-launcher@ef2c9db</a></strong></div><div>Lets you manage your pharo images and download new ones - Allow to define a short name on templates and create an image from th… · pharo-project/pharo-launcher@ef2c9db</div></blockquote>

**Comment by guillep** [comment]
I think we should wait until Pharo12 stabilises a bit here :)

**Comment by demarey** [comment]
yes, but PL uses P11, so it should work. The progress comes from PL and not the launched image.

**Comment by guillep** [comment]
@jecisc 's screenshot and his comments makes me think he got (somehow) a pharo launcher running on top of a latest P12 :)

**Comment by jecisc** [comment]
The P12 image was the one that was been downloaded by the PL :)
But I took the PL provided at the meeting about the PL-CLI. That is in P11 IIRC

**Comment by guillep** [comment]
Well I confirm I saw the bug in Launcher 3.1.1-418-ge2eb5da2

**Comment by Bajger** [comment]
Can we extend jobPresenterHeight? I think it could be because of different zoom ratio on OS: https://github.com/pharo-project/pharo-launcher/pull/608/files#diff-c99ee8b0d96b24723c910c09ac3b07cf39c7dd347af2b6c1eafa5f4481e26868R6

<blockquote><img src="https://avatars.githubusercontent.com/u/45875448?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/pharo-project/pharo-launcher/pull/608">595 general progress bar by Bajger · Pull Request #608 · pharo-project/pharo-launcher</a></strong></div><div>fixes #595


There is unrelated commit on SpConfirmDialog, but I've included that.</div></blockquote>
